.angular-cli.json

```diff
@@ -35,20 +35,12 @@
       }
     }
   ],
-  "e2e": {
```

Comment:
Why did Protractor get removed? Are we using a different solution for e2e tests? Please clarify